packages/app-builder/src/repositories/TestRunRepository.ts

```diff
@@ -156,14 +157,24 @@ const testrunRuleExecutions: TestRunRuleExecutionCount[] = [
   },
 ];
 
-export const makeGetTestRunRepository2 = () => {
+export const makeGetTestRunRepository = () => {
```

Comment:
Take care, IIUC, you are connecting the fake repos here (don't merge it)

Reply:
Yes since I need to add the endpoint to the Open Api Spec, I planned on switching back to prod adapter on the next commit